### PR TITLE
Support for Dexcom toucscreen receivers (UPLOAD-30)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses:
[UPLOAD-30]

[UPLOAD-30]: https://tidepool.atlassian.net/browse/UPLOAD-30

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tidepool-uploader",
   "productName": "tidepool-uploader",
-  "version": "2.18.0",
+  "version": "2.18.0-dexcom-touchscreen.3",
```
Comment (pazaan) [inline app/package.json]: Should this version bump to `2.19.0-dexcom-touchscreen.3`?

File: lib/drivers/dexcom/dexcomDriver.js [hunk 2]
```diff
@@ -318,12 +335,28 @@ module.exports = function (config) {
...
       }
+
+      if (header.revision >= 5){
+        // G5 touchscreen receivers do not have a predicted glucose value
+        if (softwareNumber !== 'SW10751') {
+          rec.predictedFlags &= 0xFC00;
+          rec.urgentLowSoon = hasFlag(PREDICTED_FLAGS.URGENT_LOW_SOON, rec.predictedFlags);
+          rec.predictionUnavailable = rec.predictedGlucose === 13;
+
+          // FIXME rec.predictedBackfilled = hasFlag(EGV_FLAGS.BACKFILLED, rec.predictedGlucose);
```
Comment (pazaan) [inline lib/drivers/dexcom/dexcomDriver.js]: Does this FIXME need to be addressed?

File: lib/drivers/dexcom/dexcomDriver.js [hunk 3]
```diff
@@ -318,12 +335,28 @@ module.exports = function (config) {
...
+          // according to the spec, only 10 bits are the actual glucose value
+          rec.predictedGlucose &= 0x3FF;
+        }
+
+        // FIXME rec.backfilled = hasFlag(EGV_FLAGS.BACKFILLED, rec.glucose);
```
Comment (pazaan) [inline lib/drivers/dexcom/dexcomDriver.js]: FIXME to be addressed?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tidepool-uploader",
-  "version": "2.18.0",
+  "version": "2.18.0-dexcom-touchscreen.3",
```
Comment (pazaan) [inline package.json]: Should this be `2.19.0-dexcom-touchscreen.3`?

File: app/package.json (hunk 1 above)
Comment (gniezen) [reply app/package.json]: I've built a test version for Eric to try out while this is in code review, and he discovered some issues. Once we get these resolved I'll remove the test version strings and request re-review, thanks!

**Pull request merged**